src/UI/templates/js/Input/Field/tagInput.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Wraps the BootstrapTagsInput
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+var il = il || {};
+il.UI = il.UI || {};
+il.UI.Input = il.UI.Input || {};
+(function ($) {
+    il.UI.Input.tagInput = (function ($) {
+
+
+        /**
+         *
+         * @param raw_id
+         * @param config
+         */
+        var init = function (raw_id, config) {
+            var _DEBUG = false;
+            var _CONFIG = {
+                id: '',
+                options: [],
+                selected_options: [],
+                extendable: false,
+                suggestion_starts: 1,
+                max_chars: 2000,
+                suggestion_limit: 50,
+                debug: _DEBUG,
+                allow_duplicates: false,
+                highlight: true,
+                hint: true,
+                tag_class: "label label-primary"
```

Comment:
Those default configs seem partly redundant to defaults set by PHP, why setting those here again?